pylint_nautobot/constants.py

```diff
@@ -0,0 +1,145 @@
+"""Define constants for the pylint-nautobot plugin."""
+
+# TODO Remove after dropping support for Python 3.8
+from __future__ import annotations
+
+from pylint.checkers import BaseChecker
+from pylint.typing import MessageDefinitionTuple
+
+
+class MSGS(BaseChecker):
```

Comment:
What's the purpose of having this inherit from `BaseChecker`?

Reply:
Currently, in order to align with Pylint. I see in the future the ability to inherit from MESSAGES instead of BaseChecker. I can change this back to no inheritance and just disable the pylint message that comes up.